manifests/server.pp

```diff
@@ -574,9 +580,28 @@
         ensure   => running,
         enable   => true,
         provider => 'systemd',
-        require  => [ File["/etc/openvpn/${name}.conf"], Openvpn::Ca[$ca_name] ]
+        require  => [ File["${etc_directory}/openvpn/${name}.conf"], Openvpn::Ca[$ca_name] ]
       }
     }
   }
 
+  if $::openvpn::params::namespecific_rclink {
```

Comment:
This clause does not take in account if `$::openvpn::manage_service` is `false`. At least the `service` part should be excluded.